poethepoet/task/args.py

```diff
@@ -26,6 +26,7 @@
     "positional": (bool, str),
     "required": bool,
     "type": str,
+    "multiple": (bool, int),
```

Comment:
Not sure if it's really necessary to have an int here. At least I don't know any use cases from tools I used.

Reply:
I'm not sure either. Though it seemed like a low cost feature to support, and imagined that there could be use-cases for  it.